ad/data_source_ad_group.go

```diff
@@ -16,7 +16,7 @@ func dataSourceADGroup() *schema.Resource {
 			"guid": {
```

Comment:
Is GUID still the best attribute name if it can be any of those?

Reply:
Since we're still in Tech Preview I'm in favor of the breaking change to make it more intuitive.  

@aareet What are your thoughts?

Reply:
Breaking change is ok with me too. Better now than waiting for the next major version after we go GA.

Reply:
Great ! :)